app/scripts/views/header.coffee

```diff
@@ -39,6 +41,9 @@ module.exports = class HeaderView extends View
     # Update date in all posts/comments on the page
     setInterval RefreshDateView::tick, 60000
 
+    # Check anonymous status every 500ms and indicate it with icon
+    setInterval @RefreshAnonymousStatus, 500
```

Comment:
Yes, in order to update multiple tabs to prevent case where you're actually anonymous, but in current tab button is inactive and you posting post/comment thinking you're not anonymous.